packages/salesforcedx-apex-debugger/src/messages/i18n.ts

```diff
@@ -28,5 +28,7 @@ export const messages = {
   streaming_handshake_error_text:
     'Streaming API handshake failed with the following error',
   streaming_handshake_timeout_text:
-    'Streaming API handshake failed due to a socket timeout. Check your connection and try again.'
+    'Streaming API handshake failed due to a socket timeout. Check your connection and try again.',
+  hotswap_warn_text:
+    'Apex classes & triggers should not be modified during an active Apex Debugger session.'
```

Comment:
@ruthemmanuelle Apex Debugger doesn't allow changes to Apex source code while there's an active debugger session. The server side would actually terminate the session if the user pushes Apex source. If they don't push and keep the changes locally, it could confuse their breakpoints, stacktrace, and variables. This PR is to watch .cls & .trigger files in their SFDX project, and warn them that they shouldn't change Apex source during an active debugger session. Let me know how to best convey that message in this text.

Reply:
I think there should be two message:
* a warning that notifies people that changes to class and trigger will not be honored (this can be tight to the file watch)
* a general notice that hot-swap is not support and any push will terminate the debug session (possibly at every begin of debug session and/or readme)

Reply:
@guw Regarding the second bullet point, is this sufficient? https://developer.salesforce.com/docs/atlas.en-us.eclipse.meta/eclipse/debugger_considerations.htm

Reply:
@DatGuyJonathan - How's this? 
'You can’t modify Apex classes or triggers during an Apex Debugger session. Save your changes after you’re done debugging.'

Reply:
Thanks Ruth, I'll update the text to that.